services/sasquatch/values-idfdev.yaml

```diff
@@ -0,0 +1,29 @@
+strimzi-kafka: {}
+
+influxdb:
+  ingress:
+    enabled: true
+    hostname: data-dev.lsst.cloud
+
+kafka-connect-manager:
+  influxdbSink:
+    influxdb-sink:
+      enabled: true
+
+chronograf:
+  ingress:
+    enabled: true
+    hostname: data-dev.lsst.cloud
+
+  env:
+    GENERIC_NAME: "CILogon"
+    GENERIC_AUTH_URL: https://data-dev.lsst.cloud/auth/openid/login
+    GENERIC_TOKEN_URL: https://data-dev.lsst.cloud/auth/openid/token
```

Comment:
For the record, this will use GitHub, not CILogon, on data-dev, at least for now. CILogon may be correct in the long run, though.

Reply:
Yep, I notice this after I deployed sasquatch on data-dev. I'll ask tomorrow during stand-up about which environments I could use to test CILogon.